### PR TITLE
chore: enable incrementals

### PR DESCRIPTION
**Pull request by kuisathaverat**
* enable incrementals
* remove unuseful plugin
* fix URLs

### CODE REVIEW

**Comment by kuisathaverat** [comment]
I guess I've to skip tests because there is no code

File: pom.xml [hunk 1]
```diff
@@ -5,23 +5,24 @@
...
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
 
 
     <name>Trilead API Plugin</name>
     <description>
         This plugin provides access to Trilead without having to bundle Trilead in Jenkins core
     </description>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Trilead+API+Plugin</url>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/Trilead+API+Plugin</url>
```
Comment (jglick) [inline pom.xml]: follow the redirect please, this hostname is obsolete

File: pom.xml [hunk 2]
```diff
@@ -5,23 +5,24 @@
...
     <licenses>
         <license>
             <name>The MIT license</name>
-            <url>http://www.opensource.org/licenses/mit-license.php</url>
+            <url>https://www.opensource.org/licenses/mit-license.php</url>
```
Comment (jglick) [inline pom.xml]:
```suggestion
            <url>https://opensource.org/licenses/MIT</url>
```

File: pom.xml (hunk 1 above)
Comment (kuisathaverat) [inline pom.xml]:
```suggestion
    <url>https://plugins.jenkins.io/trilead-api</url>
```

**Comment by kuisathaverat** [comment]
@jglick not sure that I can package the plugin without any java code file in it

```
[ERROR] Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:3.5:hpi (default-hpi) on project trilead-api: Error assembling hpi: /Users/inifc/src/trilead-api-plugin/target/classes isn't a directory. -> [Help 1]
```

**Comment by jglick** [comment]
You need a `src/main/resources/index.jelly`. (see https://github.com/jenkinsci/archetypes/pull/38)

**Pull request merged**